src/modules/counters.js

```diff
@@ -0,0 +1,7 @@
+const counters = () => {
+  const numOfCard = document.querySelector('.main, .items-counter').children.length;
+  const cardCounter = document.getElementById('items-counter');
+  cardCounter.innerHTML = `${numOfCard}`;
```

Comment:
You can add parentheses around the number like this `$({numOfCard})` so it will be displayed cats (6) and not cats 6